packages/protocol/contracts/thirdparty/risczero/IRiscZeroReceiptVerifier.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+/// @notice Verifier interface for RISC Zero receipts of execution.
+/// https://github.com/risc0/risc0-ethereum/blob/release-0.7/contracts/src/IRiscZeroVerifier.sol
+interface IRiscZeroReceiptVerifier {
```

Comment:
shall we copy the code from https://github.com/risc0/risc0-ethereum/blob/release-0.7/contracts/src/IRiscZeroVerifier.sol as is?

Reply:
I did that with the interface and only:
- modified the Interface name to (`IRiscZeroReceiptVerifier`) be more clear (bc. i saw it gave you some confusion why we have 2 RiscZeroVerifiers)
- removed the verify_integrity() code as it is not used, because normal `verify()` (in the implementation) calls the `verify_integrity()`